contracts/core/Registry.sol

```diff
@@ -197,13 +204,14 @@ contract Registry is AccessControl {
     /// @dev Only pending owner can claim ownership.
     function acceptIdentityOwnership(bytes32 _identityId) external {
         Identity storage identity = identitiesById[_identityId];
+        address newOwner = identityIdToPendingOwner[_identityId];
 
-        if (msg.sender != identity.pendingOwner) {
+        if (msg.sender != newOwner) {
             revert NOT_PENDING_OWNER();
         }
 
-        identity.owner = identity.pendingOwner;
-        identity.pendingOwner = address(0);
+        identity.owner = newOwner;
+        identityIdToPendingOwner[_identityId] = address(0);
```

Comment:
I usually use `delete identityIdToPendingOwner[_identityId]` but not sure if it's equivalent gas wise.

Reply:
When you use delete, it clears the storage slot associated with the given key, effectively removing the key-value pair from the mapping. The `delete` operation refunds the gas cost of the storage slot deletion. On the other hand, when you assign address(0) to a mapping value, it actually writes the value address(0) to the storage slot, consuming gas for the storage write operation.